walk-in-interview/src/main/webapp/new-job/script.js

```diff
@@ -13,12 +13,24 @@ const CurrentLocale = 'en';
  */
 import {AppStrings} from '../strings.en.js';
 
-import {getRequirementsList, setErrorMessage} from "../common-functions.js";
+import {getRequirementsList, setErrorMessage} from '../common-functions.js';
 
 const STRINGS = AppStrings['new-job'];
 const HOMEPAGE_PATH = '../index.html';
-const RESPONSE_ERROR = 'There was an error while creating' +
+const RESPONSE_ERROR = 'There was an error while creating ' +
   'the job listing, please try submitting again';
+const HOURS_PER_YEAR = 8760;
+/** Note that is an approximate value */
+const WEEKS_PER_YEAR = 52;
+const MONTHS_PER_YEAR = 12;
+
+/**
+ * Note that this is needed because in JS we can hold bigger Integer
+ * values than in Java.
+ */
+const JAVA_INTEGER_MAX_VALUE = Math.pow(2, 31) - 1;
+
+let currErrorField;
```

Comment:
this is basically a global variable. Is it possible to add a class or a tag to the form field so that we don't need that variable?